### PR TITLE
Enable Travis CI

### PR DESCRIPTION
**Pull request by hxjuneja**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js:
+    - "0.10"
+    - "0.11"
+
+before_script:
+  - sleep 5
```
Comment (earldouglas) [inline .travis.yml]: Why do we `sleep 5` here?

File: package.json [hunk 2]
```diff
@@ -15,5 +15,8 @@
   "devDependencies": {
     "mocha": "x.x.x",
     "mocha-jshint": "0.0.9"
+  },
+  "scripts": {
+    "test": "rm -f restbase && mocha"
```
Comment (d00rman) [inline package.json]: `rm -rf restbase` ? Why?

Comment (hxjuneja) [inline package.json]: restbase is the name of database and on running each test a 'restbase' file is created. deleting the restbase file deletes all the tables.

The '-f' there is used so we dont get an error if the file is not found.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+    - "0.10"
+    - "0.11"
+
+notifications:
+  email:
+    - services@wikimedia.org
+
+script: npm test
```
Comment (gwicke) [inline .travis.yml]: This shouldn't be needed, as it is the default.

Comment (gwicke) [inline .travis.yml]: This is the default, so I think it's not needed.

File: package.json (hunk 2 above)
Comment (gwicke) [inline package.json]: Could you use a more obvious database name like test_db?

Comment (hxjuneja) [inline package.json]: yeah sure

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+    - "0.10"
+    - "0.11"
```
Comment (d00rman) [inline .travis.yml:4]: put `0.12` and `iojs` there as well

**Pull request merged**